src_assets/common/assets/web/config.html

```diff
@@ -1050,33 +1055,57 @@ <h1 class="my-4">Configuration</h1>
           this.resolutions.join("," + nl + "    ") +
           nl +
           "]";
-        this.config.fps = JSON.stringify(this.fps);
+        // remove quotes from values in fps
+        this.config.fps = JSON.stringify(this.fps).replace(/"/g, "");
       },
       save() {
         this.saved = this.restarted = false;
         this.serialize();
+
+        // create a temp copy of this.config to use for the post request
+        let config = JSON.parse(JSON.stringify(this.config))
+
+        // delete default values from this.config
+        for (let key in defaultConfig) {
+          let delete_value = false
+          if (key === "resolutions" || key === "fps") {
+            let regex = /([\d]+x[\d]+)/g
+            // Use a regular expression to find each value and replace it with a quoted version
+
+            let config_value = JSON.parse(config[key].replace(regex, '"$1"')).toString()
+            let default_value = JSON.parse(defaultConfig[key].replace(regex, '"$1"')).toString()
+
+            if (config_value === default_value) {
+              delete_value = true
+            }
+          }
+
+          if (config[key] === defaultConfig[key]) {
+            delete_value = true
+          }
+
+          if (delete_value) {
+            delete config[key]
+          }
+        }
+
         fetch("/api/config", {
           method: "POST",
-          body: JSON.stringify(this.config),
+          body: JSON.stringify(config),
         }).then((r) => {
           if (r.status === 200) this.saved = true;
         });
       },
       apply() {
         this.saved = this.restarted = false;
-        this.serialize();
-        fetch("/api/config", {
```

Comment:
This was removed, since the user must already click "Save" for the "Apply" button to appear... but we will save again anyway just to be save.